protocol/v2/ssv/runner/committee.go

```diff
@@ -445,6 +446,7 @@ func (cr *CommitteeRunner) ProcessPostConsensus(ctx context.Context, logger *zap
 	if len(attestations) > 0 {
 		submissionStart := time.Now()
 		if err := cr.beacon.SubmitAttestations(attestations); err != nil {
+			// TODO: consider removing the logger.Error here
```

Comment:
duplicate err. we should consider removing it

Reply:
since this is in the context of the runner I think we can leave it even though its a duplicate.

Reply:
yeah because you wont see the Beacon logs when filtering for some duty